packages/@adobe/spectrum-css-temp/components/actiongroup/index.css

```diff
@@ -168,7 +168,8 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-ActionGroup-menu {
-  .spectrum-ActionGroup-menu-chevron {
+  /* specificity must be higher than `.spectrum-ActionButton .spectrum-Icon` */
+  svg.spectrum-ActionGroup-menu-chevron {
```

Comment:
i think previously we've handled it like
`.spectrum-ActionGroup-menu-chevron.spectrum-ActionGroup-menu-chevron` otherwise we could run into an issue if someone is doing flags and uses an img

Reply:
it's a hard coded chevron, not user provided...

Reply:
o, ok, then carry on